app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadFragment.kt

```diff
@@ -462,6 +465,12 @@ class ThreadFragment : Fragment() {
         }
     }
 
+    private fun observeClosedThread() {
+        mainViewModel.autoAdvanceTrigger.observe(viewLifecycleOwner) { listThreadsUids ->
+            tryToAutoAdvance(localSettings.autoAdvanceMode, listThreadsUids)
```

Comment:
Don't send `autoAdvanceMode` all the way down from here.
Directly use it where needed.